examples/Benchmark/Accuracy/viscous_friction/cube.obj

```diff
@@ -0,0 +1,33 @@
+# Blender3D v244 OBJ File: <memory>
+# www.blender3d.org
+o Cube

```

Comment:
To check if such a cube is not already available in `share/mesh`

Reply:
There is indeed share/mesh/cube.obj. I'm not sure how to refer to this path in the script. Using $SOFA_ROOT ?

Reply:
I think the loaders automatically check the `share` folder. Try simply `cube.obj`

Reply:
It works.